Bloon/Features/LTP/LTPSlashCommand.cs

```diff
@@ -0,0 +1,62 @@
+namespace Bloon.Features.LTP
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Bloon.Core.Database;
+    using Bloon.Variables.Roles;
+    using DSharpPlus;
+    using DSharpPlus.CommandsNext.Attributes;
+    using DSharpPlus.Entities;
+    using DSharpPlus.SlashCommands;
+
+    [SlashModuleLifespan(SlashModuleLifespan.Scoped)]
+    public class LTPSlashCommand : ApplicationCommandModule
+    {
+        private readonly BloonContext db;
+
+        public LTPSlashCommand(BloonContext db)
+        {
+            this.db = db;
+        }
+
+        [SlashCommand("ltp", "Add or remove yourself from the Looking To Play role.")]
+        public async Task LTPAsyncslash(InteractionContext ctx)
+        {
+            DiscordRole ltp = ctx.Guild.GetRole(SBGRoles.LookingToPlay);
+            DiscordMember guildUser = await ctx.Guild.GetMemberAsync(ctx.User.Id);
+            try
```

Comment:
Wrapping the entire code in a try-catch are we? Pretty sure we already have a listener handling command errors.

Reply:
whatever dude